File: Leetcode/Java/PalindromeNumber.java
/**
 * Title : 9. Palindrome Number
 * Given an integer x, return true if x is a palindrome, and false otherwise.
 */
public class PalindromeNumber{
    public boolean isPalindrome(int x) {
        if (x < 0) {
        return false;
    }
    String numStr = x+"";
    int j = numStr.length() - 1;
    for (int i = 0; i <= j; i++) {
        if (numStr.charAt(i) != numStr.charAt(j)) {
            return false;
        }
        j--;
    }
    return true;
    }
}

File: Leetcode/Javascript/palindrome-number.js
/**
 * Title : 9. Palindrome Number
 * Given an integer x, return true if x is a palindrome, and false otherwise.
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function (x) {
    if (x < 0) {
        return false;
    }
    const numStr = x.toString();
    j = numStr.length - 1;
    for (i = 0; i <= j; i++) {
        if (numStr.charAt(i) != numStr.charAt(j)) {
            return false;
        }
        j--
    }
    return true
};